backend/src/services/note.ts

```diff
@@ -0,0 +1,107 @@
+import { assert } from "console";
+import { createHash } from "crypto";
+import preSessionQuestions from "../models/preQuestionsList.json";
+import postSessionQuestions from "../models/postQuestionsList.json";
+import { Note } from "../models/notes";
+
+/*
+ * Class definition for an Answer to a question, can either be textbox or bullet boxes.
+ */
+class Answer {
+  answer: string | Array<string>;
+
+  type: string;
+
+  id: string; // hashed from question
```

Comment:
@wllmwu If you were curious, this is the ID that was mentioned on the json. Its calculated at runtime so we don't have to store it in json statically.